live-examples/js-examples/expressions/meta.json

```diff
@@ -1,5 +1,11 @@
 {
     "pages": {
+      "expressionsAdditionAssignment": {
```

Comment:
I would expect "expressionsAdditionAssignment" to sort after "expressionsAddition", and the same applies for all the others ("expressionsDivisionAssignment" after "expressionsDivision" etc). Wouldn't you, or am I missing something?

Reply:
Hm, yeah, probably. I created the files and the file explorer in VS code shows "expressions-addition-assignment.js" before "expressions-addition.js". But happy to change this. Maybe my OS or my VS code sorts weird? Or maybe even my locale? (I think I have en-US in VS Code, though),

Reply:
Just doing a sort based on code point values (not locale-sensitive) I would expect "expressions-addition-assignment.js" to sort before "expressions-addition.js", because "-" comes before ".":

But I'd expect "expressionsAddition" to sort after "expressionsAdditionAssignment", because "expressionsAddition" is a prefix of "expressionsAdditionAssignment".

In the console:

```
"expressions-addition-assignment.js" < "expressions-addition.js"
// true
"expressionsAdditionAssignment" < "expressionsAddition"
// false
```

Reply:
ahh! I shouldn't be allowed to look at my file tree then and base sorting on that :smile: